### PR TITLE
Update group-policy-analytics.md

### PR DESCRIPTION
**Pull request by mardahl**
Added a little clarification to the GPO export process (GPO not GPOs, since it can only export one at a time)
Seeing as the process does work, but does not align with what the portal instructs the user to do.
I am uncertain if the portal instructions are the best or the instructions here are best.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@mardahl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/configuration/group-policy-analytics.md [hunk 1]
```diff
@@ -62,22 +62,24 @@ This article shows you how export your GPOs, import the GPOs into Endpoint Manag
 
 - This feature is in public preview. For more information, go to [Public preview in Microsoft Intune](../fundamentals/public-preview.md).
 
-## Export GPOs as an XML file
+## Export GPO as an XML file
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/group-policy-analytics.md:65]:
```suggestion
## Export a GPO as an XML file
```

File: memdocs/intune/configuration/group-policy-analytics.md [hunk 2]
```diff
@@ -62,22 +62,24 @@ This article shows you how export your GPOs, import the GPOs into Endpoint Manag
...
 
-1. On your on-premises computer, open the `Group Policy Management` app (GPMC.msc).
-2. Expand your domain to see all the GPOs.
-3. Right-click any GPO > **Save report**:
+1. On your on-premises computer, open the `Group Policy Management` console (GPMC.msc).
+2. In the manageent console, expand your **domain name**.
+3. Then expand **Group Policy Objects** to see all the available GPOs.
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/group-policy-analytics.md:69]:
```suggestion
2. In the management console, expand your **domain name**.
3. Expand **Group Policy Objects** to see all the available GPOs.
```

File: memdocs/intune/configuration/group-policy-analytics.md [hunk 3]
```diff
@@ -62,22 +62,24 @@ This article shows you how export your GPOs, import the GPOs into Endpoint Manag
...
+4. Right-click the GPO you want to migrate and choose **Save report**:
 
     :::image type="content" source="./media/group-policy-analytics/sample-group-policy-object-save-report.png" alt-text="Open Group Policy management and save a GPO as an XML file report.":::
 
-4. Save the file to an easily accessible folder, and save it as an XML file. You'll add this file in Endpoint Manager.
+4. Select an easily accessible folder for your export and choose "Save as type" **XML File**. You'll add this file in Endpoint Manager group policy analytics.
 
-Be sure the file is less than 4 MB and has a proper unicode encoding. If the exported file is greater than 4 MB, then include fewer GPOs when you save your report from the GPMC.msc tool.
+Be sure the file is less than 4 MB and has a proper unicode encoding. 
+If the exported file is greater than 4 MB, then you must reduce the amount of settings within the selected group policy object.
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/group-policy-analytics.md:77]:
```suggestion
Make sure that the file is less than 4 MB and has a proper Unicode encoding. 
If the exported file is greater than 4 MB, you must reduce the number of settings within the selected group policy object.
```

File: memdocs/intune/configuration/group-policy-analytics.md [hunk 4]
```diff
@@ -62,22 +62,24 @@ This article shows you how export your GPOs, import the GPOs into Endpoint Manag
...
 
 ## Import GPOs and run analytics
 
 1. In the [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), select **Devices** > **Group Policy analytics (preview)**.
-2. Select **Import**, and then select your saved XML file. When you select the XML file, Intune automatically analyzes the GPO in the XML file.
+2. Select **Import**, and then select your saved XML file (you can select multiple files at once). When you select the XML file, Intune automatically analyzes the GPO in the XML file.
```
Comment (JohanFreelancer9) [inline memdocs/intune/configuration/group-policy-analytics.md:82]:
```suggestion
2. Select **Import**, and then select your saved XML file (you can select multiple files at the same time). When you select the XML file, Intune automatically analyzes the GPO in the XML file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mardahl, kindly commit the suggested changes. Many thanks.

**Comment by PRMerger15** [comment]
@mardahl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger4** [comment]
@mardahl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@mardahl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger3** [comment]
@mardahl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by PRMerger7** [comment]
@MandiOhlinger : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by MandiOhlinger — APPROVED** [review]

**Comment by MandiOhlinger** [comment]
#sign-off

**Pull request merged**